### PR TITLE
Revisit Prettier workflow

### PR DESCRIPTION
**Pull request by ann-kilzer**
## Part of #44 

Use a separate branch, since branch protection rules forbid pushes to main

### CODE REVIEW

File: pages/about.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="about">
+  <div id="about">
```
Comment (ann-kilzer) [inline pages/about.vue:2]: This is just a test that Prettier will clean up the mess 🧹

**Comment by ann-kilzer** [comment]
Here is the successful run: https://github.com/ourjapanlife/findadoc-frontend/runs/2892131413?check_suite_focus=true

File: pages/about.vue (hunk 1 above)
Comment (ann-kilzer) [reply pages/about.vue:2]: The automated fix is shown in #126

**Review by Ermish — APPROVED** [review]
LGTM

**Comment by ann-kilzer** [comment]
don't need the netlify build since this didn't change the sources

**Pull request merged**